frontend/src/utils/stringToHTML.js

```diff
@@ -0,0 +1,5 @@
+export default function stringToHTML(str) {
+  const div = document.createElement("div");
+  div.innerHTML = str;
+  return div.innerText;
+}
```

Comment:
This looks so wrong, from a security perspective. Is it possible to get clean text from the server? If not is it possible to use simply `str.replace(/<[^>]*>/g, '')`?

Also the name shouldn't be stringToHTML, it's converting from HTML to plain text.